frontend/src/components/pages/student/StudentDashboard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
does this mean once they're logged in they'll have access to the test forever? since we don't check validity of test session beyond the access code part

Reply:
a couple things here:
- whenever the browser is refreshed, the context should automatically clear, since we don't persist the student auth values to local storage). so theoretically, the user won't be able to access the assessment anymore (let me know if that seems to not be the case)
- there's a ticket in the current sprint to update the access code values so that after the time has expired, student users also shouldn't be able to access test sessions
let me know if that answers your question!

Reply:
is that validation happening in `AssessmentSummaryPage`? i.e. if they stay on that page beyond the expiration time

Reply:
that's a good point. i think theoretically if they never closed their browser they could see the test forever? but yeah, i can work on adding some sort of session expiry in a future ticket maybe

Reply:
https://www.notion.so/uwblueprintexecs/Student-Dashboard-session-expiration-000487c0e65d4210964813d5af68555c?pvs=4